tests/fixtures/modules/export/class-default-anonymous-extends.result.js

```diff
@@ -0,0 +1,6 @@
+module.exports = {
+    "index": 21,
+    "lineNumber": 1,
+    "column": 22,
+    "description": "Unexpected reserved word"
```

Comment:
well, this one is certainly valid :), there is something wrong with the code, maybe on my code :)

this is sugar for:

```
var foo = class extends bar {};
export default foo;
```